documentation/asciidoc/computers/ai/getting-started.adoc

```diff
@@ -170,6 +170,19 @@ $ sudo apt-mark unhold hailo-tappas-core hailort hailo-dkms
 
 [tabs]
 ======
+v4.19::
+To install version 4.19 of Hailo's neural network tooling, run the following commands:
++
+[source,console]
+---
```

Comment:
I think you're missing a hyphen here!

Reply:
For the result, see: https://www.raspberrypi.com/documentation/computers/ai.html#alternative-package-versions

Reply:
Good spot!! :eyes:  Fixed in #4090